src/workerd/server/server.c++

```diff
@@ -986,7 +992,7 @@ kj::Own<Server::Service> Server::makeNetworkService(config::Network::Reader conf
     tlsNetwork = ownedTlsContext->wrapNetwork(*restrictedNetwork).attach(kj::mv(ownedTlsContext));
   }
 
-  return kj::heap<NetworkService>(globalContext->headerTable, timer, entropySource,
+  return kj::refcounted<NetworkService>(globalContext->headerTable, timer, entropySource,
```

Comment:
Any reason not to use `kj::Rc<...>` and `kj::rc<...>` with this?

Reply:
Mentioned in a commit message:

> I realize it would be even better to go all the way to `kj::Rc` here, but that would create a lot more noise, also requiring a coordinated internal change, so I'm punting that for the moment.